cmd/opampsupervisor/supervisor/commander/commander.go

```diff
@@ -67,7 +76,8 @@ func (c *Commander) Start(ctx context.Context) error {
 	c.cmd.Stdout = logFile
 	c.cmd.Stderr = logFile
 
-	c.doneCh = make(chan struct{})
+	c.doneCh = make(chan struct{}, 1)
+	c.exitCh = make(chan struct{}, 1)
```

Comment:
I think we should just create these once on startup then drain them above. That way we can let the caller figure out multicasting.